src/Chapter1.hs

```diff
@@ -631,8 +638,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = 
+    let digit1 = mod (abs n) 10
+        digit2 = mod (div (abs n) 10) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod m 10
div m 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod m 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Hmm...
I fail to see how this would be simpler - I need to apply the modulo-division by 10 to the result of the first division whereas `divMod` only applies it to the original value albeit as `div` and as `mod` at the same time.